PyPI tarball missing a test file

Hi David,

I was updating the OpenBSD package of shrinkray and ran into the following:
```
============================= test session starts ==============================
platform openbsd7 -- Python 3.13.11, pytest-9.0.2, pluggy-1.6.0
rootdir: /usr/local/pobj/shrinkray-26.2.4.1/shrinkray-26.2.4.1
configfile: pyproject.toml
plugins: trio-0.8.0, hypothesis-6.138.14, typeguard-4.4.4, asyncio-1.3.0, cov-7.0.0
asyncio: mode=Mode.STRICT, debug=False, asyncio_default_fixture_loop_scope=None, asyncio_default_test_loop_scope=function
collected 669 items / 13 errors

==================================== ERRORS ====================================
_____________ ERROR collecting tests/test_byte_reduction_passes.py _____________
ImportError while importing test module '/usr/local/pobj/shrinkray-26.2.4.1/shrinkray-26.2.4.1/tests/test_byte_reduction_passes.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../../lib/python3.13/importlib/__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/test_byte_reduction_passes.py:19: in <module>
    from tests.helpers import assert_reduces_to, direct_reductions, reduce_with
E   ModuleNotFoundError: No module named 'tests.helpers'
```

The problem is that `tests/helpers.py` (which *is* in git) isn't in the release tarball hosted on PyPI:

```
$ tar ztvf shrinkray-26.2.4.1.tar.gz | grep help
$
```

Presumably just needs to be added to some manifest somewhere.

(Might be worth checking for other missing files too)

Cheers

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,0 +1,5 @@
+include pytest.ini
+include justfile
+
+recursive-include tests *.py *.svg *.jinja2
+recursive-include scripts *.py

